core/src/main/scala/org/broadinstitute/dsde/rawls/entities/datarepo/DataRepoBigQuerySupport.scala

```diff
@@ -176,53 +293,15 @@ trait DataRepoBigQuerySupport {
    * @param entityQuery user-supplied query criteria
    * @return the object to pass to BQ to execute a query
    */
-  def queryConfigForQueryEntities(dataProject: String, viewName: String, entityType: String, entityQuery: EntityQuery): QueryJobConfiguration = {
+  def queryConfigForQueryEntities(dataProject: String, viewName: String, entityType: String, entityQuery: EntityQuery): QueryJobConfiguration.Builder = {
```

Comment:
this seemingly unrelated change (and similar that follow) allows us to set maximumBytesBilled centrally